docs/developing-prometheus-rules-and-grafana-dashboards.md

```diff
@@ -298,8 +301,10 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') + {
   _config+:: {
     namespace: 'monitoring',
   },
-  grafanaDashboards+:: {
```

Comment:
this one is intentionally this way, as it's the convention for the monitoring-mixins